### ISSUE TITLE
VOD recover

### ISSUE THREAD

**Issue by ElenaPoGG** [TwitchRecover/TwitchRecover#356]
Hey! 
ive been trying to recover this VOD: https://streamscharts.com/channels/elenapog_/streams/46083340572
its a really really important vod and idk how to use the app Please.

**Comment by ghost** [comment]
@ElenaPoGG Doesn't exist.

**Comment by Fikus25** [comment]
Hello, I want to ask if is possible to download this VOD, I'm trying to download it(using Youtube-DLG / directly via TwichRecover application) but nothing happens. Is possible that TwitchRecover was able to get the m3u8 file but the VOD parts itself are not available anymore?
Here is the link for the m3u8: https://dgeft87wbj63p.cloudfront.net/12848ee429a84547af1a_himtheoldboy_45791243452_1658400295/chunked/index-dvr.m3u8

Thanks

**Comment by ghost** [comment]
@Fikus25 

Enter M3U8 Link: https://dgeft87wbj63p.cloudfront.net/12848ee429a84547af1a_himtheoldboy_45791243452_1658400295/chunked/index-dvr.m3u8
0 of 2066 Segments are valid

Vod is not playable.

**Comment by Fikus25** [comment]
@ItIckeYd Thanks for confirmation.

**Comment by Fikus25** [comment]
Hello, I'm trying to recover the deleted VOD from 11-11-2022, is it possible to get this? I have also downloaded the last version of VodRecovery tool but no result. The tool should be able to get the deleted vods too so am I doing something wrong?

Links are:
https://streamscharts.com/channels/selick1/streams/46288110780
https://twitchtracker.com/selick1/streams/46288110780

**Comment by ghost** [comment]
@Fikus25 

Vod just doesn't exist unfortunately. The tool can only recover what twitch doesn't delete.

**Comment by Fikus25** [comment]
@ItIckeYd Ok, so the tool can't recover VOD that's been deleted by streamers themselves?

**Comment by ghost** [comment]
@Fikus25 

If you can prove that is the case then its possible but I think its premature to come to that conclusion based off one video alone. 

There are many reasons a vod may not exist.. As to the exact reason why this one doesn't exist, I'm not sure anyone would know unless they have a good understanding of twitch's backend and their vod deletion process.

**Comment by Fikus25** [comment]
@ItIckeYd yes, this vod was deleted by the streamer, he said it during the stream.

**Comment by ghost** [comment]
@Fikus25 I'd have to find more instances where this is the case to come to a conclusion but for this vod... we can assume that is the reason.
